en/hubs_privacy_notice.md

```diff
@@ -42,7 +42,7 @@ __Firefox account information.__ A Hubs subscription requires a Firefox account,
 
 __Location information.__ Mozilla receives your IP address from your Firefox account when you sign up for and use the service. We use the IP address in order to approximate your location because Mozilla VPN is currently only available in certain countries.
 
-__Payment information.__ When you subscribe to Hubs, you will send payment through one of our third-party payment providers: Stripe, Apple, PayPal, or Google Pay. Mozilla receives a record of your account (including your billing address and the last four digits of your payment method) and the status of your account’s subscription. Mozilla does not store your full payment details.
+__Payment information.__ When you subscribe to the Services, you will send payment through one of our third-party payment providers: [Stripe](https://stripe.com/privacy), [Apple](https://www.apple.com/legal/privacy/data/en/apple-pay/), [PayPal](https://www.paypal.com/us/legalhub/privacy-full), or [Google Pay](https://payments.google.com/payments/apis-secure/u/0/get_legal_document?ldo=0&ldt=privacynotice&ldl=en). Mozilla receives a record of your account (including your billing address and the last four digits of your payment method) and the status of your account’s subscription, but does not store your full payment details. The payment providers may also collect personal transactional data or device information via cookies and similar technologies to operate and improve their service.  You can learn more about how these companies use and process your data by reviewing their respective privacy policies, linked above.
```

Comment:
Nit: we don't use double spaces after a period in these docs (it doesn't have any effect).

On a person note: these links are going to be a burden when it comes to maintain this document, and a problem if we ever decide to localize it (they either point on the US, or English version of documents).

Reply:
Thanks for the note on spacing.  

The localization point is a good one.  It may make sense to modify the language and just direct people to review the appropriate Privacy Policy on their provider's website.  I'll circle back to you on this later today.  If I wanted to modify the language I included in the prior request, how would I do that?  Would we need to close this out and have me submit a new request?

Reply:
You can edit the file in the branch that GitHub automatically generated

1. Go to files: https://github.com/mozilla/legal-docs/pull/1872/files
2. Click the 3 dots button (bottom right) on the file you want to edit. That should lead you to [this URL](https://github.com/aroybalreid/legal-docs/edit/patch-4/en/hubs_privacy_notice.md?pr=%2Fmozilla%2Flegal-docs%2Fpull%2F1872).
3. Make the changes and commit at the bottom. That should update this PR automatically.

P.S. I've also set mafeld as reviewer, as my team is only doing the administrative work for this repository ;-)